app/src/test/java/ch/hikemate/app/model/profile/ProfileRepositoryFirestoreTest.kt

```diff
@@ -170,7 +195,7 @@ class ProfileRepositoryFirestoreTest {
 
     repository.getProfileById(
         "1",
-        onSuccess = { p -> fail("Success callback should not be called") },
+        onSuccess = { assertNull(it) },
```

Comment:
Question: Why have you changed this? `onSuccess` should never have been called in the first place.

Reply:
Yes sorry I really have no Idea why.
I must've changed something and then forgot.
I just put it back.